bindata/v3.11.0/openshift-apiserver/deploy.yaml

```diff
@@ -134,12 +134,12 @@ spec:
           periodSeconds: 5
           timeoutSeconds: 10
           successThreshold: 1
-          failureThreshold: 1
+          failureThreshold: 3
         startupProbe:
           httpGet:
             scheme: HTTPS
             port: 8443
-            path: healthz
+            path: livez
```

Comment:
Shouldn't this follow the liveness probe by excluding the etcd check?

Reply:
I don't think so. This probe is executed only once and without etcd we cannot start the server.

Reply:
It may not make much difference. Both readyz and livez are gated on post-start hook completion, and we know that many of those require informer syncs. What's the upside to including the etcd check in the startup probe? If etcd is unavailable for some reason, we may as well continue waiting for it to come back instead of failing the startup probe and restarting.

Reply:
My reasoning is that the startup probe is executed only once and is used to know if the container has successfully started. If the kas  cannot reach etcd during startup, it may indicate a fundamental issue that should block the startup until it is resolved.